### ISSUE TITLE
"Do not include" not always being honored in generator

### ISSUE THREAD

**Issue by randoogle** [keepassxreboot/keepassxc#5462]
When I'm dolling our temporary passwords for a particular system, I need to exclude several characters (otherwise they'll get interpreted). Lately, I've noticed that the password generator doesn't always seem to honor the "Do not include" option, and I'll start getting passwords with the excluded characters in them.
After I cut and paste the list of offending characters back into the same field, it appears to start working again.

**Comment by droidmonkey** [comment]
What version of keepassxc are you using

**Comment by droidmonkey** [comment]
I could not replicate this, you will need to provide more detail on the exact circumstances you observe this situation.

**Comment by randoogle** [comment]
I'm running the flatpak version of 2.6.1 revision 9a35bba
It's doing it again today. I'm not sure how specific I can be, as I'd prefer not to specify which characters aren't allowed this particular system for security purposes. I guess I can try and tinker with different character sets to see if I can reproduce it with specific ones.
It's doing it again today though. My workaround seems to work consistently.
*update*
I did a little more experimenting, and I found that it happens after opening the app. I have almost all the character types selected (specifically A-Z a-z 0-9 #$%... <*+!?= {[()]]}  ).  I put in "()" for the "Do not include" option. Of course it works when I first put those characters in the "Do not include" field, but after closing Keepass and opening again, sure enough, there's parenthesis in the passwords (until I modify the "Do not include" field again).
